### ISSUE TITLE
Need documentation around local Dockerfile to local Containerfile and related command usage

### ISSUE THREAD

**Issue by davidthewatson** [containers/toolbox#914]
**Is your feature request related to a problem? Please describe.**
I've used toolbox on the silverblue series of Fedora releases since 32. I'm using 35 currently.

I'd like to have better documentation for using Containerfile from the command line with toolbox.

The [readme](https://github.com/containers/toolbox/tree/main/images/fedora/f35) does not define or explain the Containerfile in the directory it occupies.

1. Define that Silverblue treats Containerfile like Dockerfile with the Fedora-specific limits imposed (since this directory is f35).
2. Define the `build, create, run` use case, but for the silverblue toolbox stack. Alternatively, alias the the 3 CLI commands below into a single `toolbox .` command or something like that.
3. Stretch goal: Put a distro roadmap in the images directory so that the lower-level READMEs aren't just copies of the root readme [here](https://github.com/containers/toolbox/blob/main/README.md), but apply to the specific Containerfiles where they are found.
  
**Describe the solution you'd like**
I'd like to have documentation similar to this:

https://github.com/jelford/webdev-toolbox#local-development

In other words, how to use the tools to read the Containerfile locally, not the repository use case which is invoked by some of the tooling silently without warning.

**Describe alternatives you've considered**
I searched  google and github to find the page linked above that describes the simple use case I was looking for around local Containerfile wokflow with podman and toolbox:

```
// git clone this repo, then
cd images/fedora/f35
podman build -t mytb . // the Containerfile
toolbox create -i mytb -c mytb // use the image
toolbox enter mytb 
```

**Additional context**
I'm happy to make a small documentation pull request to get this started. I love what toolbox has done and is doing for the OCI container initiative and I'd be glad to help, where you're open to additional help.

**Comment by debarshiray** [comment]
I am a bit confused. :)

Currently, Toolbox doesn't support using a Containerfile (or a Dockerfile) directly.  It looks for an OCI image.  It might be good to teach `toolbox create` to directly use a Containerfile for custom images, but currently that's not the case.

The image definitions under the `images` sub-directory are meant to be the sources to the OCI images that Toolbox uses by default.  They are there to make it easier for Toolbox contributors to see what the images look like, and be able to quickly make modifications to them and try them out.  Otherwise one would have to go hunting in the infrastructure of different distributions, and that would quickly become problematic as the number of supported distributions start to grow.

That said, it would definitely be nice to tweak the `README.md` files for the OCI images to be something that more specifically describes the images, and not the project as a whole.

By the way, https://github.com/jelford/webdev-toolbox looks nifty. :)

**Comment by debarshiray** [comment]
Duplicate of https://github.com/containers/toolbox/issues/879

**Comment by debarshiray** [comment]
Closing.

Please feel free to leave a comment or re-open if you don't think it's a duplicate.
